enlighten/Controller.py

```diff
@@ -218,6 +218,7 @@ def __init__(self,
 
         # default to Scope view
         self.page_nav.set_main_page(common.Pages.SCOPE)
+        self.page_nav.set_view_scope()
```

Comment:
### Background info

`self.page_nav.set_main_page`
sets PageNav's stack_main index
which corresponds to QtObj:comboBox_view
![image](https://github.com/WasatchPhotonics/ENLIGHTEN/assets/124081765/e2c815e2-a979-4bf2-ba24-8cdfdf227a51)
This is the dropdown with options Scope, Settings, Hardware

`self.page_nav.set_view_scope`
sets PageNav's combo_view index
self.combo_view.setCurrentIndex
which corresponds to QtObj:stackedWidget_low
![image](https://github.com/WasatchPhotonics/ENLIGHTEN/assets/124081765/77afb158-a355-4a03-b735-6f0546cf590d)
This is the widget that contains each of those screens listed in the dropdown.

These should not be independently settable outside of PageNavigation.

The solution is to put `self.page_nav.set_main_page` calls within PageNav's `set_view...` methods.

And actually that is already done.

### Conclusion

Line 220 is now redundant and should be removed

Reply:
Sounds good.  Please test on a non-Raman spectrometer to confirm.

Reply:
Actually, I just tested this and...it's not redundant.  It doesn't actually work without that line.